### PR TITLE
Feat add iss and aud claims

### PR DESCRIPTION
**Pull request by like-a-bause**
# Description

Adds the ability to define config: Sessions.Issuer which gets used in the JWT iss claim, if set.
The field is optional.

# Implementation

# Tests

# Todos

* add config option to docs
* 

# Additional context

### CODE REVIEW

File: backend/config/config.go [hunk 1]
```diff
@@ -391,6 +391,8 @@ type Session struct {
 	EnableAuthTokenHeader bool   `yaml:"enable_auth_token_header" json:"enable_auth_token_header" koanf:"enable_auth_token_header" split_words:"true"`
 	Lifespan              string `yaml:"lifespan" json:"lifespan" koanf:"lifespan"`
 	Cookie                Cookie `yaml:"cookie" json:"cookie" koanf:"cookie"`
+	// Issuer optional String to be used in the jwts iss claim. Can be a uri or string
```
Comment (FreddyDevelop) [inline backend/config/config.go:394]:
```suggestion
	// Issuer optional string to be used in the jwt iss claim.
```

**Review by FreddyDevelop — COMMENTED** [review]
Some thing that came around, we also do not set the `audience`, do we want to add this option too? Some JWT libraries require the claim, e.g. https://github.com/cfworker/cfworker/tree/main/packages/jwt

Also the docs for the `iss` claim is still missing.

**Comment by like-a-bause** [comment]
audience might be a good addition. Should this be a separate config parameter or automatically set to e.g. relyingParty.Id?

**Comment by FreddyDevelop** [comment]
Hmm, maybe default is `relyingParty.id` but you can set it yourself through a config parameter. One thing to have in mind, is that `audience` can be a string array, do we want to support that, or is it always a single string, [see](https://www.rfc-editor.org/rfc/rfc7519.html#page-9:~:text=In%20the%20general%20case%2C%20the%20%22aud%22%20value%20is%20an%20array%20of%20case%2D%0A%20%20%20sensitive%20strings%2C%20each%20containing%20a%20StringOrURI%20value.%20%20In%20the%0A%20%20%20special%20case%20when%20the%20JWT%20has%20one%20audience%2C%20the%20%22aud%22%20value%20MAY%20be%20a%0A%20%20%20single%20case%2Dsensitive%20string%20containing%20a%20StringOrURI%20value.).

**Comment by like-a-bause** [comment]
@FreddyDevelop suggested that we remove the default audience whenever a user sets the config parameter. So the user has full control.

File: backend/session/session_test.go [hunk 2]
```diff
@@ -36,7 +37,10 @@ func TestGenerator_Generate(t *testing.T) {
 func TestGenerator_Verify(t *testing.T) {
 	sessionLifespan := "5m"
 	manager := test.JwkManager{}
-	cfg := config.Session{Lifespan: sessionLifespan}
+	cfg := config.Config{
+		Session: config.Session{Lifespan: sessionLifespan},
+	}
+	//cfg := config.Session{Lifespan: sessionLifespan}
```
[the remark this answers is not in the document]
Comment (like-a-bause) [reply backend/session/session_test.go]: 👍

**Review by FreddyDevelop — APPROVED** [review]
LGTM

**Pull request merged**